mirgecom/multiphysics/thermally_coupled_fluid_wall.py

```diff
@@ -1381,6 +1608,10 @@ def coupled_ns_heat_operator(
             fluid_grad_temperature, wall_grad_temperature,
             fluid_boundaries, wall_boundaries,
             interface_noslip=interface_noslip,
+            interface_radiation=interface_radiation,
+            wall_emissivity=wall_emissivity,
+            sigma=sigma,
+            ambient_temperature=ambient_temperature,
```

Comment:
If the function is deprecated, I wouldn't updated it with radiation.

Reply:
https://github.com/illinois-ceesd/mirgecom/pull/893/commits/31ce3239f59a815d21a079a87c047e9a48e37431